src/DSFML/Audio/SoundBuffer.cpp

```diff
@@ -68,17 +68,17 @@ void sfSoundBuffer_destroy(sfSoundBuffer* soundBuffer)
     delete soundBuffer;
 }
 
-DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename)
+DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename, size_t length)
 {
-    return (soundBuffer->This.saveToFile(filename))? DTrue: DFalse;
+    return (soundBuffer->This.saveToFile(std::string(filename, length)))? DTrue: DFalse;
 }
 
 const DShort* sfSoundBuffer_getSamples(const sfSoundBuffer* soundBuffer)
 {
     return soundBuffer->This.getSamples();
 }
 
-size_t sfSoundBuffer_getSampleCount(const sfSoundBuffer* soundBuffer)
```

Comment:
I actually have it as size_t on purpose as I personally think size_t is better to use than ulong. I have been meaning to bring this up on the forums for a long time since I think it is a better design. We have to cast it somewhere anyway since we need it to be a size_t when we make a slice. I figured it might as well be here.